.circleci/config.yml

```diff
@@ -2,26 +2,41 @@ version: 2 # use CircleCI 2.0
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/node:4.8.2 # ...with this image as the primary container; this is where all `steps` will run
-      - image: mongo:3.4.4 # and this image as the secondary service container
+      - image: circleci/node:14.8-stretch # ...with this image as the primary container; this is where all `steps` will run
+        environment:
+          DB_CONNECTION: mongodb://localhost
+      - image: mongo:4.4-bionic # and this image as the secondary service container
+      - image: redis:6-alpine # and this image as the 3rd service container
     steps: # a collection of executable commands
       - checkout # special step to check out source code to working directory
       - run:
           name: update-npm
-          command: "sudo npm install -g n"
+          command: "sudo npm install -g n mocha"
       - run:
           name: update-npm
           command: "sudo n stable"
-      - restore_cache: # special step to restore the dependency cache
-          # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
-          key: dependency-cache-{{ checksum "server/package.json" }}
       - run:
           name: install-npm-wee
           command: cd server && npm install
-      - save_cache: # special step to save the dependency cache
-          key: dependency-cache-{{ checksum "server/package.json" }}
-          paths:
-            - server/node_modules
-      - run: # run tests
-          name: test
-          command: cd server && npm test
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.6.1
+      - run:
+          name: Wait for redis
+          command: dockerize -wait tcp://localhost:6379 -timeout 1m
```

Comment:
I don't think we need redis at the moment tbh, but we can leave it here in case we add it later on